apps/generics/actions.py

```diff
@@ -13,8 +19,7 @@ class Action:
     icon_class: str = None
     extra_context: dict = None
     required_permissions: list = dataclasses.field(default_factory=list)
-
-    display_condition: callable = None
+    display_condition: Callable[[HttpRequest, Context], bool] = display_condition
```

Comment:
these models and functions are used in other tables as well do I don't think you can default them to functions that assume the record is a `Tag`. Instead you must find the the actions are being used and pass in the display condition function there. For tags I think it is the `apps.annotations.tables.TagTable` class

Reply:
[9b6c01f](https://github.com/dimagi/open-chat-studio/pull/440/commits/9b6c01f8dfe8bc7fb726f86447dd75f4ca53cb2b)